projects/storefrontlib/src/cms-components/myaccount/update-email/update-email-form/update-email-form.component.html

```diff
@@ -63,7 +63,7 @@
           }}"
           class="form-control"
           [class.is-invalid]="isNotValid('password')"
-          autocomplete="off"
+          autocomplete="new-password"
```

Comment:
It seems to be some kind of `workaround` approach :) afaik `autocomplete` value shouldn't be different than `on/off`. 

Did you try to add `autocomplete="off"` into entire `<form>` tag instead of inputs? If it would work i think will be more elegant solution, but if not we can keep this.